LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019 Kristoffer Carlsson <kristoffer.carlsson@juliacomputing.com>
```

Comment:
Hm, I mean I see that this code is mainly by you, but would it make sense to leave me in here, or do we just say, its in the history?

Reply:
Maybe something like: Kristoffer, Simon, Luca, and other contributors? Just to be comprehensive.

Reply:
That sounds fair!

Reply:
I would appreciate that, as I put a lot of effort on PackageCompiler.jl development.

Reply:
Definitely ;) That I was alone in the license was wrong to begin with!

Reply:
Can someone make a suggestion here so it can just be accepted? Otherwise, we absolutely must add the names. Also let's add `Julia Computing` to the mix please.

Reply:
Yeah I'll make a suggestion

Reply:
Sorry, didn't think about this, it was just the default that got generated by PackageCompiler. Let's merge all names in here :)